tests/test_validator.py

```diff
@@ -32,6 +36,48 @@ def set_validator(validator_name):
     os.environ["NBFORMAT_VALIDATOR"] = validator_name
 
 
+@pytest.mark.parametrize("validator_name", VALIDATORS)
+def test_should_warn(validator_name):
+    """Test that a v4 notebook witout id emit a warning"""
+    set_validator(validator_name)
+    with TestsBase.fopen("test4.5.ipynb", "r") as f:
+        nb = read(f, as_version=4)
+
+    del nb.cells[3]["id"]
+    assert nb.cells[3].get("id") is None
```

Comment:
Isn't that a little bit redundant ? I'm not sure what error we could introduce in the code that would trigger this `assert` ?